CMakeLists.txt

```diff
@@ -139,6 +139,14 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
 #    set(DEFAULT_MALLOC "jemalloc")
 endif()
 
+hpx_option(HPX_WITH_THREAD_STACKOVERFLOW_DETECTION
+  BOOL
+  "Enable stackoverflow detection for HPX threads/coroutines. (default: ON)"
+  ON ADVANCED)
```

Comment:
The issue we had back then was to supporting multi configuration builds. As those don't exist in any of the *nix ecosystems (or does WSL change the picture?), I would consider this a non problem for this particular feature.